src/component/ProductsFilter/ProductsFilter.jsx

```diff
@@ -54,6 +100,13 @@ const ProductsFilter = () => {
             id='seeds'
             name='checkbox-seeds'
             type='checkbox'
+            checked={filterState.filterByCategories.includes("Super Seeds")}
+            onChange={() =>
+              filterDispatch({
+                type: "FILTER_BY_CATEGORY",
+                payload: "Super Seeds",
+              })
+            }
           />
           Seeds
         </label>
```

Comment:
suggestion: the categories jsx is repetitive. You can map it from an array of appropriate data.

Reply:
I was trying to do it but i think i was doing some mistake so moved back to this code.I will try to do it again after authentication part is completed